### PR TITLE
added Like delay

### PR DESCRIPTION
**Pull request by c0d3x27**
let's say you want to randomize the time between each like, correct? If so, this is it.

### CODE REVIEW

File: instapy/instapy.py [hunk 1]
```diff
@@ -1168,6 +1168,7 @@ def like_by_tags(self,
                      interact=False,
                      randomize=False,
                      media=None):
+                     sleep_delay=60
```
Comment (CharlesCCC) [inline instapy/instapy.py]:
```
sleep_delay=60): 
```

???

Comment (c0d3x27) [reply instapy/instapy.py]: im sorry, i just fixed that.

**Comment by timgrossmann** [comment]
@Banginpadr Thank you very much for your contribution!
It seems like you only changed it for the `like_by_tag` method, do you thin you could also adjust it for the other features that are missing the `sleep_time` propoerty?
Would be awesome.

Otherwise, please check the failed checks for the error that says that `sleep_delay` at some point in your code is not defined

Best 
Tim

**Comment by Banginpadr** [comment]
@timgrossmann sure as soon i get back home ;)

**Comment by sionking** [comment]
@timgrossmann I have some delay limits builtin the global settings, do you want me to pr this ?

**Comment by timgrossmann** [comment]
@sionking Sounds good and most likely exactly what this does only for all the features instead of only `like_by_tag`.

Thanks for the offer @sionking

**Comment by CharlesCCC** [comment]
please make sure update the READ.me wiki as well 👍

**Comment by jtrieuDev** [comment]
How can I use this version to add delay to my like ?

**Comment by sionking** [comment]
pr this:
https://github.com/timgrossmann/InstaPy/issues/3046#issuecomment-425639615

**Pull request closed without merging**